pagopa/io-app
# chore: [IA-91] Contextual help event

## Short description
Emit an event when the user taps on the help button, respecting user's privacy.

## List of changes proposed in this pull request
- Transform `BaseHeaderComponent` in a functional component to allow the use of hooks (namely `useSelector`)
- Separate `ContextualInfo` from `ContextualHelp` since they don't interact
- simplify `ts/utils/navigation.ts` based on the available types from `react-navigation`

## How to test
In any screen, press the `?` button on the top right to open the contextual menu. You will be able to notice the new event in the dashboard.
